python/python/tests/test_async_client.py

```diff
@@ -9367,3 +9367,48 @@ async def test_script_binary(self, redis_client: TGlideClient):
         assert (
             await redis_client.invoke_script(script, keys=[key2]) == "value2".encode()
         )
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_script_large_keys_no_args(self, request, cluster_mode, protocol):
+        redis_client = await create_client(
```

Comment:
Please rename `redis_client` to `glide_client`

Reply:
all of the redis_client references would be changed in a separate PR